module/VuFind/src/VuFind/RateLimiter/RateLimiterManager.php

```diff
@@ -211,6 +211,26 @@ public function check(EventInterface $event): array
                     $result['allow'] = false;
                     $result['message'] = $this->getTooManyRequestsResponseMessage($event, $result);
                     $result['presentTurnstileChallenge'] = ($priorTurnstileResult === null);
+                    if (
+                        $result['presentTurnstileChallenge'] &&
+                        ($this->config['Policies'][$policyId]['turnstileIgnoredRateLimiterSettings'] ?? false)
+                    ) {
+                        $turnstileIgnoredLimiter = ($this->rateLimiterFactoryCallback)(
+                            $this->config,
+                            $policyId,
+                            $this->clientIp,
+                            $this->userId,
+                            'turnstileIgnoredRateLimiterSettings'
+                        );
+                        $turnstileIgnoredLimit = $turnstileIgnoredLimiter->consume(1);
+                        if (!$turnstileIgnoredLimit->isAccepted()) {
+                            $this->verboseDebug('No more ignoring the challenge.');
+                            $this->turnstile->setResult($policyId, $this->clientIp, false);
+                            $result['presentTurnstileChallenge'] = false;
+                        } else {
+                            $this->verboseDebug('Turnstile ignored limit down to ' . $turnstileIgnoredLimit->getRemainingTokens());
```

Comment:
Looks like you've got a line length issue here:

```suggestion
                            $this->verboseDebug(
                                'Turnstile ignored limit down to ' . $turnstileIgnoredLimit->getRemainingTokens()
                            );
```

Reply:
This is what happens when I step away for a quick lunch -- you are too fast :)